### DISCUSSION TITLE
pogo pin ideas

### DISCUSSION THREAD

**Discussion by justme-1968** [stevenbennett/DIYson#27] [Ideas]
just saw your latest video introducing the pogo pins and two things came to mind:

- maybe it would help to move the pogo pins nearer to the center of rotation instead of ‘far’ out on the side of the carriage. this would minimize the relative movement of the track and the pins to each other.   

-  instead of the tracks being on the front and top sides they could be on the left and right sides so that they can be even closer to the center of the carriage. this would also minimize the distance between the pogo pins. 

combining both ideas could allow mounting the pins somewhere inside the carriage which would make them even invisible. maybe the pins could be mounted to both sides of a small pcb that can be clipped into the carriage center between the upright and the arm. this would also eliminate the cable.

**Comment by jhogendorn** [comment]
Just adding in some thoughts re pogo pins after the most recent video.

In no particular order:

1. Obviously reducing the play in the mechanism is a good goal. Are you tensioning the bearing wheels using the eccentric spacer n uts that 3d printers use? You could also put a ptfe/hdpe (https://www.aliexpress.com/item/1005001335382182.html) threaded piece  in the track and bolt to that through the plate which will _greatly_ reduce the amount of possible flex. 2x per axis and it will be much much more rigid.
2. The PCB should be made with 'hard gold' finish, or `ENIG`. This is necessary for it to cope with the wear of the motion.
3. The rolling ball contact pins, while much lower stroke will also be important for longevity. The non rolling pins will slowly wear away the track
4. Pin positioning should be moved to be in the middle axis of the plate where torque will be the least. the tracks should face the 'inside' faces.
5. The pin assemblies themselves can be preloaded so you get travel distance in the design. For example, if the plate was made with a leaf spring part for the pins to sit on that would preload and allow more flexibility. That way you're not relying on the small travel distance on the pin itself.
6. You are only transmitting power. If you use conductive wheels, you can use the body of the lamp as ground and only run a VCC rail in the pcb. Which means you can upsize your pins and do things like use a 2 pair rolling ball assembly on the track. If you have a scenario where it can be manipulated to a point where no wheel has contact, you're a wizard harry.
7. If you are using the body as ground, you can do this pogo pin trick at the base as well for the rotation of the vertical extrusion. (you can do it without but its easier if you use the body as ground)
8. As you saw in the video, firmly locating the track in the rail is important. But you may find that the best approach is to have the pogo pin assembly actually attached to the rail its reading from instead of relying on pin travel and preload. if the pin assembly is attached to the target extrusion in a way that slides easily, the pin contact is constant and the slop in the overall mechanism is taken up by wiring. If you grounded the chassis, you could even put the pin pcb in the channel itself and have it ride on a strip of conductive copper tape.
9. Rather than 3d printing or cncing the main bracket, consider using a design that layers pcb. 3 2mm boards = 6mm plate. Fab is fast, you can do a lot with silkscreens, you can solder in threads, lets you integrate the circuitry seamlessly into the part itself, pcb's are very rigid and precise and cheap. If you're smart you layer 3 identical pcb's from the panel to save even more costs.
10. Capacitor banking the led is a smart idea, you have a lot of space in the head, doing a larger array would make sense. You just need enough capacity to bridge the worst case period of voltage drop, which you were almost getting with just the one. put 4-6 on there on the head pcb. Its a good idea in general anyway. If you wanted you could even just put a battery in the head and the power supply is mainly to keep the battery charged instead of directly powering the light. means you will get light during loss of power for a while also. If you're space restricted you can get rechargeable button cells, 2032 is an option but LIR2450 gives a fair amount of capacity, and you can throw a few cells in there if needbe. Alternatively a 1632 would be smaller than the extrusion (20mm) and you could 'stack' multiple of them to make essentially a custom height battery with whatever space is available in the head. Aliexpress sells cheap battery management pcbs that can handle all the charging / switching management for you for a few dollars.
11. Last point, if the plate is made of pcbs, and you have a ptfe slider in the rail, you can use a ball end screw (https://www.ebay.com.au/itm/303897909273) in the pcb through a solder in nut (https://core-electronics.com.au/smt-solderable-standoff-nuts-m3-x-3mm-10-pack.html) (or you can use a rivnut) to go through the ptfe slider and touch the underlying track/copper tape. so you're grounding the circuit through the frame, this can be your VCC transmission and rigidity in one.

Hope this is all useful :)

**Comment by Spireneusz** [comment]
Capacitors draw fair amount of current when charging and can greatly reduce life of supertrack and pogo pins when used alone. Use them with resistor connected in series that will reduce current spikes.
